### PR TITLE
AWS Terraform App Access, DB listeners variables

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 1]
```diff
@@ -269,6 +269,26 @@ that people use to connect to your Teleport cluster, so choose wisely.
 
 This must be a subdomain of the domain you chose for [`route53_zone`](#route53\_zone) above.
 
+### add\_wildcard\_route53\_record
+  
+Setting `export TF_VAR_add_wildcard_route53_record"`
```
Comment (ptgott) [inline docs/pages/setup/deployments/aws-terraform.mdx:274]: Should there be an assignment here?

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 2]
```diff
@@ -269,6 +269,26 @@ that people use to connect to your Teleport cluster, so choose wisely.
...
+
+Used to enable Application Access for subdomains of the Teleport Web Address. A wildcard entry for the public facing
```
Comment (ptgott) [inline docs/pages/setup/deployments/aws-terraform.mdx:276]:
```suggestion
Used to enable Application Access for subdomains of the Teleport Proxy Service's public web address. A wildcard entry for the public-facing 
```

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 3]
```diff
@@ -269,6 +269,26 @@ that people use to connect to your Teleport cluster, so choose wisely.
 
 This must be a subdomain of the domain you chose for [`route53_zone`](#route53\_zone) above.
 
+### add\_wildcard\_route53\_record
+  
+Setting `export TF_VAR_add_wildcard_route53_record"`
+
+Used to enable Application Access for subdomains of the Teleport Web Address. A wildcard entry for the public facing 
+domain will be set in Route 53 , e.g. *.teleport.example.com, to point to the Teleport load balancer.  For ACM a wildcard
```
Comment (ptgott) [inline docs/pages/setup/deployments/aws-terraform.mdx:277]:
```suggestion
domain will be set in Route 53, e.g., `*.teleport.example.com`, to point to the Teleport load balancer. For ACM a wildcard 
```

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 4]
```diff
@@ -269,6 +269,26 @@ that people use to connect to your Teleport cluster, so choose wisely.
 
 This must be a subdomain of the domain you chose for [`route53_zone`](#route53\_zone) above.
 
+### add\_wildcard\_route53\_record
+  
+Setting `export TF_VAR_add_wildcard_route53_record"`
+
+Used to enable Application Access for subdomains of the Teleport Web Address. A wildcard entry for the public facing 
+domain will be set in Route 53 , e.g. *.teleport.example.com, to point to the Teleport load balancer.  For ACM a wildcard 
+certificate is included if this is set true.  The LetsEncrypt automatically includes a wildcard subdomain as part of its issuance.
```
Comment (ptgott) [inline docs/pages/setup/deployments/aws-terraform.mdx:278]:
```suggestion
certificate is included if this is set to `true`. Let's Encrypt automatically includes a wildcard subdomain in certificates that it issues.
```

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 5]
```diff
@@ -269,6 +269,26 @@ that people use to connect to your Teleport cluster, so choose wisely.
...
+
+### enable\_mongodb\_listener
+
+A port (27017) is enabled on the Network Load Balancer that connects to the Teleport MongoDB listener port.  Required for MongoDB database access.
```
Comment (ptgott) [inline docs/pages/setup/deployments/aws-terraform.mdx:282]:
```suggestion
Port `27017` is enabled on the Network Load Balancer that connects to the Teleport MongoDB listener port. Required for MongoDB database access.
```

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 6]
```diff
@@ -269,6 +269,26 @@ that people use to connect to your Teleport cluster, so choose wisely.
...
+  
+### enable\_mysql\_listener
+
+A port (3036) is enabled on the Network Load Balancer that connects to the Teleport MySQL listener port.  Required for MySQL connections.
```
Comment (ptgott) [inline docs/pages/setup/deployments/aws-terraform.mdx:286]:
```suggestion
Port `3036` is enabled on the Network Load Balancer that connects to the Teleport MySQL listener port. Required for MySQL connections.
```

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 7]
```diff
@@ -269,6 +269,26 @@ that people use to connect to your Teleport cluster, so choose wisely.
...
+  
+### enable\_postgres\_listener
+
+A port (5432) is enabled on the Network Load Balancer that connects to the Teleport PostgreSQL listener port.  Required for PostgreSQL connections.
```
Comment (ptgott) [inline docs/pages/setup/deployments/aws-terraform.mdx:290]:
```suggestion
Port `5432` is enabled on the Network Load Balancer that connects to the Teleport PostgreSQL listener port. Required for PostgreSQL connections.
```

File: docs/pages/setup/deployments/aws-terraform.mdx (hunk 1 above)
Comment (stevenGravy) [reply docs/pages/setup/deployments/aws-terraform.mdx]: addressed @ptgott

**Pull request closed without merging**